start_json_bourne.bat

```diff
@@ -0,0 +1,21 @@
+REM @echo off
+
+set MYDIRBLOCK=%~dp0
+call %MYDIRBLOCK%stop_json_bourne.bat
+set CYGWIN=nodosfilewarning
+call %MYDIRBLOCK%..\..\..\config_env_base.bat
+
+set EPICS_CAS_INTF_ADDR_LIST=127.0.0.1
+set EPICS_CAS_BEACON_ADDR_LIST=127.255.255.255
+
+set IOCLOGROOT=%ICPVARDIR%/logs/ioc
+for /F "usebackq" %%I in (`cygpath %IOCLOGROOT%`) do SET IOCCYGLOGROOT=%%I
+
+set JSON_BOURNE_CONSOLEPORT=9012
```

Comment:
Again hard coded. Are we sure this port is free? I know the proc server assigns fixed ports to some processes, but is it recorded anywhere who uses what to avoid duplication?